src/include/simeng/Pool.hh

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstddef>
+#include <cstdint>
```

Comment:
Why was this library needed if no other changes were made?

Reply:
My simeng wont build without it :'( I will remove it from the PR though